semgrep_output_v1.atd

```diff
@@ -1050,6 +1050,10 @@ type cli_output_extra = {
 
     (* EXPERIMENTAL: since: 1.37.0 *)
     ~skipped_rules: skipped_rule list;
+
+    (* SCA subproject resolution results 
+     * EXPERIMENTAL: since: 1.123.0 *)
```

Comment:
is this comment just a convention, or do we have some tooling that can parse these annotations?

Reply:
just convention